Hangsman/game.py

```diff
@@ -0,0 +1,42 @@
+from hangman import Hangman
+from word_pick import Word_Picker
+
+def start():
```

Comment:
understand its a small game, so i immediately understand what start() is trying to do. 
however in general, we should write a bit more descriptive name, like start_hangman_game